src/desktop/version.cpp

```diff
@@ -183,19 +152,18 @@ std::string os_version()
 			return ver;
 		}
 	}
-#endif
-
+		
 	//
-	// POSIX uname version.
+	// POSIX uname version fallback.
 	//
-
+		
 	utsname u;
-
+		
 	if(uname(&u) != 0) {
 		ERR_DU << "os_version: uname error (" << strerror(errno) << ")\n";
 	}
-
-	return formatter() << u.sysname << ' '
+		
+	return formatter() 	<< u.sysname << ' '
```

Comment:
Please don't try to align the fields (they are not aligned; note that the following lines are indented with tabs).